swarm_dial.go

```diff
@@ -457,3 +460,16 @@ func (s *Swarm) dialAddr(ctx context.Context, p peer.ID, addr ma.Multiaddr) (tra
 	// success! we got one!
 	return connC, nil
 }
+
+func combineAddressDialErrors(defaultError error, dialErrors ...error) error {
```

Comment:
FYI, we have some other packages that use https://github.com/hashicorp/go-multierror.

Reply:
Cool, I have tried to find something like this with grep but failed (only custom NewMultiError at core/corerepo/gc.go). Will change here to hashicorp/go-multierror.

Reply:
Yeah, I brought this up because I'd like to switch over to some consistent multi-error package.